src/frontend/Komponenter/Behandling/Vurdering/StyledVurdering.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import navFarger from 'nav-frontend-core';
+
+export const StyledVurderingLesemodus = styled.div`
```

Comment:
```suggestion
export const VurderingLesemodus = styled.div`
```

Reply:
evt. `Container`

Reply:
Vi bruker vel `Styled.....` som komponentnavn ganske gjennomgående i vår kodebase. Jeg synes i hvert fall det gjør det tydlig hva komponenten gjør.

Hva tenker du?

Reply:
Ikke veldig viktig for meg altså, ser bare ikke helt gevinsten ved endring :)

Reply:
Tilbakemeldingen er basert på retningslinjene [fra dette repoet](https://github.com/navikt/familie). Ser at vi ikke følger de slavisk, nei 😛 

Personlig så synes jeg at koden blir mer lesbar jo mer presise navn komponentene får og at `
Styled`-prefixen ikke er særlig presis. I tillegg bidrar den til unødvendige lange komponentnavn.

Dette er jo ikke en sak som er særlig viktig i seg selv men jeg tror at kortere komponentnavn bidrar til mer lesbar kode in the long run :)

Reply:
Hehe, har du sett. Helt enig i at ørten 'StyledDiv' blir stygt og unødvendig. Men personlig liker jeg å forstå at det er en styled component utifra navnet, og føler derfor at Styled... blir ganske presist. Det gjør meg heller ikke så mye med lange navn :P 

Tanker @blommish @charliemidtlyng ?

Reply:
Jeg svikter, men PRØVER å finne andre navn enn `Styled...` på komponenter jeg lager nå.

Er det en Wrapper/Container ? Er det en `KnappMedLuftUnder`? Er det `VurderingLeseModusGrid` ?

Ofte er styles relativt begrensede og konkrete - i de tilfellene er det fint om det navnet gjenspeiler hva som skjer..?